clientd3d/config.c

```diff
@@ -83,25 +87,30 @@ static char INIShow[]        = "Show";
 static char INIMaxX[]        = "MaxX";
 static char INIMaxY[]        = "MaxY";
 
-static char comm_section[]   = "Comm";  /* Section for comm stuff in INI file */
+static char comm_section[]   = "Comm";  /* Section for comm stuff in meridian.ini */
 static char INIPort[]        = "Port";
 static char INIRedialDelay[] = "RedialDelay";
 static char INIHostname[]    = "Hostname";
 static char INISockPort[]    = "SocketPort";
 static char INIServerNum[]   = "ServerNumber";
 static char INIDomainFormat[] = "Domain";
 
-static char users_section[]  = "Users";  /* Section for dealing with other users */
+static char users_section[]  = "Users";  /* Section for dealing with other users in meridian.ini */
 static char INIDrawNames[]   = "DrawNames";
 static char INIIgnoreAll[]   = "IgnoreAll";
 static char ININoBroadcast[] = "NoBroadcast";
 static char INIIgnoreList[]  = "IgnoreList";
 
-static char special_section[] = "Special";  /* Section for hidden stuff in INI file */
+static char special_section[] = "Special";  /* Section for hidden stuff in in meridian.ini */
```

Comment:
You don't need to put the filename in every comment here; it's redundant.  Instead beef up the comment at the top, making it clear that there are two different ini files.